Makefile

```diff
@@ -53,7 +53,7 @@ $(BUILDDIR)/linux/.config: $(BUILDDIR)/linux $(CONFIG)
 	make ARCH=$(ARCH) KCONFIG_ALLCONFIG=$$(pwd)/$(CONFIG) -C $(BUILDDIR)/linux allnoconfig
 
 $(BUILDDIR)/$(KERN_IMAGE): $(BUILDDIR)/linux $(BUILDDIR)/linux/.config
-	make -j$(nprocs) ARCH=$(ARCH) CROSS_COMPILE=$(CROSS_COMPILE) -C $(BUILDDIR)/linux
+	make -j$$(nproc) ARCH=$(ARCH) CROSS_COMPILE=$(CROSS_COMPILE) -C $(BUILDDIR)/linux
```

Comment:
I'm not sure I understand why the double `$` is needed

Reply:
@gkelly Nice catch, sorry it took so long! I think the alternative approach with the _shell_ function would also improve readability. Could you make the change so we can get the PR merged finally? 🙂

Reply:
Updated the diff with `$(shell nproc)` and pushed. Tested locally with `docker build --build-arg BUILD_ARCH=aarch64 .`, and saw my machine using only `nproc` cores.